File: 1029-two-city-scheduling/1029-two-city-scheduling.py
class Solution:
    def twoCitySchedCost(self, costs: List[List[int]]) -> int:
        extra = [];
        min_cost=0;
        n = len(costs)//2;
        for A,B in costs:
            extra.append(B-A);
            min_cost +=A;
        extra.sort();
        for i in range(n):
            min_cost +=extra[i];
        return min_cost;
        

File: 1029-two-city-scheduling/1029-two-city-scheduling.cpp
class Solution {
public:
    int twoCitySchedCost(vector<vector<int>>& costs) 
    {
        vector<int> extra;
        int n = costs.size()/2;
        
        int min_cost = 0;
        
        for(auto it : costs)
        {
            min_cost += it[0];
            extra.push_back(it[1] - it[0]);
        }
        
        
        sort(extra.begin(), extra.end());
        
        for(int i=0; i<n; i++)
        {
            min_cost += extra[i];
        }
        return min_cost;
        
    }
};